pkg/synchromanager/clustersynchro/cluster_synchro.go

```diff
@@ -302,8 +301,9 @@ func (s *ClusterSynchro) setSyncResources() {
 			if _, ok := s.storageResourceSynchros.Load(storageGVR); ok {
 				continue
 			}
+			resourceConfig := config.storageConfig
 
-			resourceStorage, err := s.storage.NewResourceStorage(config.storageConfig)
+			resourceStorage, err := s.storage.NewResourceStorage(resourceConfig)
```

Comment:
Because it is not used elsewhere, there is no need to assign additional values to variables